# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/HyperTextAccessible.cpp
# Commit: 1bf4b290248b
# Full Hash: 1bf4b290248bb7ff5ae1c8bfe48016e2f03f5b54
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-07-12 09:33:27
# Regressor Bug: 1757121
# File Overlap Count: 1
# Description:
#   Bug 1757121 part 1: Allow nsAccessibilityService::MarkupAttribute/MarkupAttributes to take an Accessible. r=morgan
#   
#   Previously, these methods took a DOM nsIContent, making it impossible for them to work for RemoteAccessible.
#   GetMarkupMapInfoForNode has been renamed to GetMarkupMapInfoFor and has overloads for nsIContent and Accessible.
#   MarkupAttribute is a template which can take either, as we still need the nsIContent version elsewhere.
# ==============================================================================

diff -r 919ec20e9cef -r 1bf4b290248b accessible/generic/HyperTextAccessible.cpp
--- a/accessible/generic/HyperTextAccessible.cpp	Tue Jul 12 02:44:31 2022 +0000
+++ b/accessible/generic/HyperTextAccessible.cpp	Tue Jul 12 02:45:56 2022 +0000
@@ -1406,7 +1406,7 @@
   }
 
   if (HasOwnContent()) {
-    GetAccService()->MarkupAttributes(mContent, attributes);
+    GetAccService()->MarkupAttributes(this, attributes);
     if (mContent->IsMathMLElement()) SetMathMLXMLRoles(attributes);
   }
 